nieuwe home page

braver opened the pull request:
Zie hier voor een up-to-date demo:
https://braver.github.io/alan-platform.github.io/

braver committed:
more space for text in the hero image

braver committed:
new hero image and call to action button

braver committed:
fix JS error

braver committed:
disable natural bulbs and apenheul logos for now

braver committed:
add logo assets

braver committed:
alternative layout for call 2 action button

braver committed:
more transparent header

braver committed:
add cache busters for js assets

braver committed:
improve the home hero

braver committed:
clean up

braver committed:
move the scroll hint and fade it in

braver committed:
add a link to the medium post on the homepage

braver committed:
fix the mobile menu

braver commented:
Ik zet de link naar het artikel even op de homepage... Ik weet niet of de precieze plek de beste is zo, maar zo staat ie er wel op. Op zich is het wel een mooie manier om vanaf de site te linken naar posts op medium, in plaats van alles op de site zelf te doen. Zo heb je meer kans dat het artikel gevonden wordt, en levert het ook weer links op naar de site wat weer goed is voor scores in de zoekmachines.

<img width="688" alt="Screen Shot 2019-10-15 at 22 02 24" src="https://user-images.githubusercontent.com/2543659/66865429-7c898c00-ef97-11e9-87e0-5163a26d9c10.png">

rickvanderarend commented:
Ik had nog niet gezien dat deze uitstond! Ziet er al wel mooi uit, eerlijk gezegd. Is dit zogezegd 'af'?

braver commented:
Ja, deze is ready to go, afhankelijk misschien van waar je de link naar het artikel wil hebben. Maar daar kun je misschien zelf ook nog mee schuiven.

Als je meer artikelen hebt, dacht ik zelf dat je de nieuwste misschien wel in als button kan zetten (zoals dat ene artikel nu) en dan de rest er in een eenvoudig lijstje er onder. Maar daar kunnen we ook later nog even naar kijken.

rickvanderarend commented:
Ok, thanks!

Wat betreft de customers van Kjerner:

- misschien dat we daar iets minder schermruimte per klant moeten pakken en
er meer een 'tegelvloer' van moeten maken, waar je op kunt klikken om
details te bekijken (als die er zijn). Is dat wat? Ik wil er nu nog een
aantal aan toevoegen, maar het wordt een hele lange lijst zo en sommigen
hebben eigenlijk niet veel details, denk ik.
- wat betreft de eindklanten en partners van Alan: daar is nu 1 lijstje van
op de homepage. Is daar een ook ergens een partnersoverzicht te maken?

Groet,
Rick

On Fri, 18 Oct 2019 at 07:04, Koen Lageveen <notifications@github.com>
wrote:

> Ja, deze is ready to go, afhankelijk misschien van waar je de link naar
> het artikel wil hebben. Maar daar kun je misschien zelf ook nog mee
> schuiven.
>
> Als je meer artikelen hebt, dacht ik zelf dat je de nieuwste misschien wel
> in als button kan zetten (zoals dat ene artikel nu) en dan de rest er in
> een eenvoudig lijstje er onder. Maar daar kunnen we ook later nog even naar
> kijken.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/alan-platform/alan-platform.github.io/pull/13?email_source=notifications&email_token=AADYO4YSYISZPOCIDOECIILQPE7XFA5CNFSM4JBBHCY2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEBSUH5Y#issuecomment-543507447>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AADYO4ZAET4MCUXJH2QF2QTQPE7XFANCNFSM4JBBHCYQ>
> .
>

braver commented:
> er meer een 'tegelvloer' van moeten maken

Ja, top idee

> Is daar een ook ergens een partnersoverzicht te maken?

Je kan altijd gewoon een pagina maken met logo en daar onder een verhaaltje, dat hoeft niet per se apart vormgegeven te worden. Dat is sowieso een goed begin en dan kan ik altijd nog even kijken om alles precies netjes te maken. Als die pagina er goed uitziet kunnen we dat strookje op de homepage laten linken naar de overzichtspagina.

rickvanderarend commented:
Overigens vind ik de witte tekst rechtsonder wat slecht leesbaar zo op dat
hek.

En nog een idee: kunnen we juist niet iets van die foto behouden als je
naar beneden scrollt of naar een andere pagina gaat? Ik merk dat het
allemaal wat saai wordt op de andere pagina's, als de topbalk ook wit is.


On Fri, 18 Oct 2019 at 07:04, Koen Lageveen <notifications@github.com>
wrote:

> Ja, deze is ready to go, afhankelijk misschien van waar je de link naar
> het artikel wil hebben. Maar daar kun je misschien zelf ook nog mee
> schuiven.
>
> Als je meer artikelen hebt, dacht ik zelf dat je de nieuwste misschien wel
> in als button kan zetten (zoals dat ene artikel nu) en dan de rest er in
> een eenvoudig lijstje er onder. Maar daar kunnen we ook later nog even naar
> kijken.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/alan-platform/alan-platform.github.io/pull/13?email_source=notifications&email_token=AADYO4YSYISZPOCIDOECIILQPE7XFA5CNFSM4JBBHCY2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEBSUH5Y#issuecomment-543507447>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AADYO4ZAET4MCUXJH2QF2QTQPE7XFANCNFSM4JBBHCYQ>
> .
>

gjkunst commented:
@braver zou je een overlay willen toevoegen aan de afbeelding? En zou je de tekst van de knoppen `18 px` willen maken? De afbeelding leidt nu erg af vind ik, en de knoppen vind ik niet duidelijk.

Voorbeeld met overlay: 

<img width="1428" alt="transp" src="https://user-images.githubusercontent.com/2600713/67083196-03f31d00-f19b-11e9-9bf9-07114d459ddb.png">

rickvanderarend commented:
Lijkt me niet handig als je nu van meerdere kanten potentieel
conflicterende instructies krijgt, Koen. Wij gaan eerst even overleggen.

Groet,
Rick

On Fri, 18 Oct 2019 at 11:31, gjkunst <notifications@github.com> wrote:

> @braver <https://github.com/braver> zou je een overlay willen toevoegen
> aan de afbeelding? En zou je de tekst van de knoppen 18 px willen maken?
> De afbeelding leidt nu erg af vind ik, en de knoppen vind ik niet duidelijk.
>
> Voorbeeld met overlay:
>
> [image: Uploading transp.png…]
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/alan-platform/alan-platform.github.io/pull/13?email_source=notifications&email_token=AADYO44HFMZ2Q3NH3E3FBJTQPF7AVA5CNFSM4JBBHCY2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEBTSKVI#issuecomment-543630677>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AADYO4YKV23OSYAC76C65RTQPF7AVANCNFSM4JBBHCYQ>
> .
>

rickvanderarend commented:
Hoi Koen,

Ok, done. Samenvatting:

- De tekst rechtsonder is nu niet goed leesbaar. Het is wel hetgeen waar
het om draait: de slogan die moet blijven hangen.
- De foto is op een wat groter scherm niet van voldoende kwaliteit, zegt
Gerben. Heb jij een verkleinde variant online gezet, Koen?

De suggestie van Gerben om er een overlay overheen te leggen was een poging
om beide problemen in 1 op te lossen.

Het is een mogelijkheid, maar ik vind het wel zonde dat het sprankelende
van de foto er een beetje af gaat. Zie jij nog andere mogelijkheden, Koen?
BV:
- Is er nog een kwalitatief betere variant van de foto online te krijgen
(mogelijk alleen bij grotere schermen)?
- Een overlay over de foto die alleen onder die tekst staat kan daar ook
voldoende helpen.

Verder is de tekst in het menu wel wat erg weinig zichtbaar in eerste
instantie, vanwege de donkere tekst op de donkere achtergrond. En hij is
ook wat klein. Dat kan wel beter, denken we.

En dan tenslotte: valt er denk je nog iets te doen met die foto onder de
menubalk bij andere gelegenheden dan de eerste homepage, helemaal bovenaan?
Hij is verder overal wit en dat is wel een beetje erg saai, eerlijk gezegd.

Groet,
Rick

On Fri, 18 Oct 2019 at 11:37, Rick van der Arend <rvanderarend@gmail.com>
wrote:

> Lijkt me niet handig als je nu van meerdere kanten potentieel
> conflicterende instructies krijgt, Koen. Wij gaan eerst even overleggen.
>
> Groet,
> Rick
>
> On Fri, 18 Oct 2019 at 11:31, gjkunst <notifications@github.com> wrote:
>
>> @braver <https://github.com/braver> zou je een overlay willen toevoegen
>> aan de afbeelding? En zou je de tekst van de knoppen 18 px willen maken?
>> De afbeelding leidt nu erg af vind ik, en de knoppen vind ik niet duidelijk.
>>
>> Voorbeeld met overlay:
>>
>> [image: Uploading transp.png…]
>>
>> —
>> You are receiving this because you commented.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/alan-platform/alan-platform.github.io/pull/13?email_source=notifications&email_token=AADYO44HFMZ2Q3NH3E3FBJTQPF7AVA5CNFSM4JBBHCY2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEBTSKVI#issuecomment-543630677>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AADYO4YKV23OSYAC76C65RTQPF7AVANCNFSM4JBBHCYQ>
>> .
>>
>

rickvanderarend commented:
Overigens: hij wordt wel steeds mooier hoor! Maar we willen deze versie dan
wel graag goed uitrollen.

On Fri, 18 Oct 2019 at 14:04, Rick van der Arend <rvanderarend@gmail.com>
wrote:

> Hoi Koen,
>
> Ok, done. Samenvatting:
>
> - De tekst rechtsonder is nu niet goed leesbaar. Het is wel hetgeen waar
> het om draait: de slogan die moet blijven hangen.
> - De foto is op een wat groter scherm niet van voldoende kwaliteit, zegt
> Gerben. Heb jij een verkleinde variant online gezet, Koen?
>
> De suggestie van Gerben om er een overlay overheen te leggen was een
> poging om beide problemen in 1 op te lossen.
>
> Het is een mogelijkheid, maar ik vind het wel zonde dat het sprankelende
> van de foto er een beetje af gaat. Zie jij nog andere mogelijkheden, Koen?
> BV:
> - Is er nog een kwalitatief betere variant van de foto online te krijgen
> (mogelijk alleen bij grotere schermen)?
> - Een overlay over de foto die alleen onder die tekst staat kan daar ook
> voldoende helpen.
>
> Verder is de tekst in het menu wel wat erg weinig zichtbaar in eerste
> instantie, vanwege de donkere tekst op de donkere achtergrond. En hij is
> ook wat klein. Dat kan wel beter, denken we.
>
> En dan tenslotte: valt er denk je nog iets te doen met die foto onder de
> menubalk bij andere gelegenheden dan de eerste homepage, helemaal bovenaan?
> Hij is verder overal wit en dat is wel een beetje erg saai, eerlijk gezegd.
>
> Groet,
> Rick
>
> On Fri, 18 Oct 2019 at 11:37, Rick van der Arend <rvanderarend@gmail.com>
> wrote:
>
>> Lijkt me niet handig als je nu van meerdere kanten potentieel
>> conflicterende instructies krijgt, Koen. Wij gaan eerst even overleggen.
>>
>> Groet,
>> Rick
>>
>> On Fri, 18 Oct 2019 at 11:31, gjkunst <notifications@github.com> wrote:
>>
>>> @braver <https://github.com/braver> zou je een overlay willen toevoegen
>>> aan de afbeelding? En zou je de tekst van de knoppen 18 px willen
>>> maken? De afbeelding leidt nu erg af vind ik, en de knoppen vind ik niet
>>> duidelijk.
>>>
>>> Voorbeeld met overlay:
>>>
>>> [image: Uploading transp.png…]
>>>
>>> —
>>> You are receiving this because you commented.
>>> Reply to this email directly, view it on GitHub
>>> <https://github.com/alan-platform/alan-platform.github.io/pull/13?email_source=notifications&email_token=AADYO44HFMZ2Q3NH3E3FBJTQPF7AVA5CNFSM4JBBHCY2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEBTSKVI#issuecomment-543630677>,
>>> or unsubscribe
>>> <https://github.com/notifications/unsubscribe-auth/AADYO4YKV23OSYAC76C65RTQPF7AVANCNFSM4JBBHCYQ>
>>> .
>>>
>>

braver commented:
> De tekst rechtsonder is nu niet goed leesbaar.

Klopt, die rails zit een beetje in de weg. Ik ga even kijken wat we kunnen doen. Overlays zijn een mogelijkheid, maar grote vlakken of een verloop over de foto leggen is zonde van de foto dus dat gaan we niet doen.

>  De foto is op een wat groter scherm niet van voldoende kwaliteit

Op een klein scherm is ie ook weer wat te groot. Er zijn niet zoveel mensen met beeldschermen zo groot als die van Gerben. Nu is ie mooi geschikt voor een full HD scherm. Als we op een punt zijn dat we overall tevreden zijn ga ik een reeks verschillende uitsnedes maken om hem te optimaliseren voor verschillende devices, maar als we met alles nog aan het schuiven zijn is dat de moeite niet waard.

De een z'n saai is de ander z'n fris ;). Het is op andere pagina's juist niet de bedoeling dat die balk heel zwaar leunt op de pagina. Met die donkere balk was dat wel het geval. Ik denk dat bezoekers van de site veel minder last hebben van dat idee van saaiheid omdat ze geïnteresseerd zijn in de content. Daar moet de aandacht ook naar uitgaan, die balk hebben ze dan wel gezien en mag saai zijn. Alle pagina's zijn wel lappen met tekst, dus als je daar screenshots e.d. aan toe zou voegen helpt dat ook.

braver committed:
improve main menu readability

braver committed:
improve contrast and readability of slogan

braver committed:
remove previous hero image

braver commented:
Een beetje spelen met contrast en schuiven met de foto levert volgens mij al heel wat op. 

<img width="1440" alt="Screen Shot 2019-10-18 at 22 22 16" src="https://user-images.githubusercontent.com/2543659/67125757-c4a5ea00-f1f5-11e9-903f-2fefb147599f.png">

braver committed:
update the uncropped hero image

braver committed:
change the article button into a general link

ie. link to an overview of articles on Medium
that way you don't have to manage a list on this
website or update it that often

braver committed:
put hydro on the homepage
